lib/daru/dataframe.rb

```diff
@@ -2061,8 +2061,58 @@ def access_row_tuples_by_indexs *indexes
       res
     end
 
+    def summarize(options={})
+      colmn_value = []
+      index_tuples = Array(@index).uniq
+      # if length of index_tuple is less than 2 then add one colmn of index,
```

Comment:
I am getting something unwanted when this kind of case comes : 

```
irb(main):008:0> df = Daru::DataFrame.new({b: [11,12,13,14,15], a: [101,102,103,104,105],
irb(main):009:2*   c: [11,22,33,44,55]},
irb(main):010:1*   order: [:a, :b, :c],
irb(main):011:1*   index: [[:k], [:k], [:k], [:l], [:l]])
=> #<Daru::DataFrame(5x3)>
       a   b   c
   k 101  11  11
     102  12  22
     103  13  33
   l 104  14  44
     105  15  55
irb(main):012:0> df[:a][:k]
TypeError: no implicit conversion of Daru::MultiIndex into Integer
```

 I am expecting this 

```
=> #<Daru::Vector(3)>
       a
   0 101
   1 102
   2 103
```

It works when levels.size > 1 : 

```
irb(main):013:0> df = Daru::DataFrame.new({b: [11,12,13,14,15], a: [101,102,103,104,105],
irb(main):014:2*   c: [11,22,33,44,55]},
irb(main):015:1*   order: [:a, :b, :c],
irb(main):016:1*   index: [[:k, 1], [:k,2], [:k,3], [:l,4], [:l,55]])
=> #<Daru::DataFrame(5x3)>
           a   b   c
   k   1 101  11  11
       2 102  12  22
       3 103  13  33
   l   4 104  14  44
      55 105  15  55
irb(main):017:0> df[:a][:k]
=> #<Daru::Vector(3)>
       a
   1 101
   2 102
   3 103
```